test/tests/functional/pbs_job_array_comment.py

```diff
@@ -84,11 +84,11 @@ def test_job_array_comment(self):
             ATTR_comment: (MATCH_RE, 'Job Array Began at .*')
         }
         self.server.expect(JOB, attr, id=jid, attrop=PTL_AND)
-        self.server.expect(JOB, {ATTR_comment: 'Subjob failed'},
+        self.server.expect(JOB, {ATTR_comment: 'Not Running: PBS Error: Execution server rejected request and failed'},
```

Comment:
you need to breakup this line , as pep8 allows 80 chars per line

Reply:
Fixed it, surprisingly autopep8 doesn't show this warning